docs/Config.md

```diff
@@ -4,14 +4,18 @@
 
 The location of the user config will differ depending on your OS. You can open it via lazydocker by opening the application, clicking on the 'project' panel at the top left and pressing 'o' (or pressing 'e' if your files open in vim).
 
-Changes to the user config will only take place after closing and re-opening lazydocker
+Changes to the user config will only take place after closing and re-opening lazydocker.
 
 ### Locations:
 
-- OSX: `~/Library/Application Support/jesseduffield/lazydocker/config.yml`
-- Linux: `~/.config/lazydocker/config.yml`
+Lazydocker follows the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) meaning that the config will be placed in `XDG_CONFIG_HOME` if the variable is found.
+
+- UNIX(if `XDG_CONFIG_HOME` environment variable is found): `$XDG_CONFIG_HOME/lazydocker/config.yml`
+- OSX(if `XDG_CONFIG_HOME` environment variable is **not** found): `~/Library/Application Support/jesseduffield/lazydocker/config.yml`
+- Linux(if `XDG_CONFIG_HOME` environment variable is **not** found): `~/.config/lazydocker/config.yml`
```

Comment:
Imo, this should take place as a note. after. even if readers "stop reading", instead of bloating.

Reply:
I agree: this should just say that if the XDG_CONFIG_HOME env var is set, we use $XDG_CONFIG_HOME/lazydocker/config.yml, then the rest can be stated as fallbacks without being as verbose